### PR TITLE
upgrade ethers.js to 5.4.1

### PR DESCRIPTION
**Pull request by perski6**

### CODE REVIEW

**Review by marekkirejczyk — APPROVED** [review]

**Comment by marekkirejczyk** [comment]
Upgraded in https://github.com/EthWorks/Waffle/pull/572

File: waffle-chai/package.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
   },
   "dependencies": {
     "@ethereum-waffle/provider": "^3.4.0",
```
Comment (VIPOJIKOP) [inline waffle-chai/package.json:41]: Ieyyiyz. 4iiiu6hh

Comment (VIPOJIKOP) [reply waffle-chai/package.json:41]: _oxx8 55 6_ox7@d__

File: waffle-cli/package.json [hunk 2]
```diff
@@ -49,7 +49,7 @@
     "@ethereum-waffle/compiler": "^3.4.0",
     "@ethereum-waffle/mock-contract": "^3.3.0",
     "@ethereum-waffle/provider": "^3.4.0",
```
Comment (VIPOJIKOP) [inline waffle-cli/package.json:51]: .

**Pull request closed without merging**